.github/workflows/welcome_message.yml

```
@@ -0,0 +1,18 @@
+name: 'Welcome New Contributors'
+
+on:
+  issues:
+    types: [opened]
+  pull_request_target:
+    types: [opened]
+
+jobs:
+  welcome-new-contributor:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Greet the contributor'
+        uses: garg3133/welcome-new-contributors@v1.2
```

From a security perspective, I would recommend you pin to a specific commit of the Action to ensure its tag (e.g. `v1.2`) is not maliciously swapped out from under you without giving you the opportunity to review it:

```suggestion
        uses: garg3133/welcome-new-contributors@a38583ed8282e23d63d7bf919ca2d9fb95300ca6
```